app/src/test/java/nu/yona/app/api/manager/impl/AuthenticateManagerImplTest.java

```diff
@@ -34,211 +31,240 @@
-        String otpFromresponseCode = (String) otpFromresponse.getCode();
-        if (otpFromresponseCode.equals(correctOtpCode.getCode())) {
-            listener.onDataLoad(getMockedUser());
-        } else {
-            ErrorMessage errorMessage = new ErrorMessage();
-            errorMessage.setMessage("Invalid Pass Code");
-            errorMessage.setCode("400");
-            listener.onError(errorMessage);
-        }
-    }
-
-    private void handleGetUserResponse(String url, String yonaPassword ,DataLoadListener listener){
-        listener.onDataLoad(getMockedUser());
-    }
-
-
-    private void setUpMockedAuthNetworkImplMethods(){
-        Mockito.doAnswer((Answer) invocation -> {
-            handleUserRegResponse(invocation.getArgument(0),invocation.getArgument(1),invocation.getArgument(2),invocation.getArgument(3),invocation.getArgument(4));
-            return null;
-        }).when(authenticateNetworkImplMock).registerUser(ArgumentMatchers.any(String.class),
-                ArgumentMatchers.any(String.class),ArgumentMatchers.any(RegisterUser.class),
-                ArgumentMatchers.any(Boolean.class), ArgumentMatchers.any(DataLoadListener.class));
-
-        Mockito.doAnswer((Answer) invocation -> {
-            handleVerifyMobileResponse(invocation.getArgument(0),invocation.getArgument(1),invocation.getArgument(2),invocation.getArgument(3));
-            return null;
-        }).when(authenticateNetworkImplMock).verifyMobileNumber(ArgumentMatchers.any(String.class),
-                ArgumentMatchers.any(String.class), ArgumentMatchers.any(OTPVerficationCode.class),
-                ArgumentMatchers.any(DataLoadListener.class));
-
-        Mockito.doAnswer((Answer) invocation -> {
-            handleGetUserResponse( invocation.getArgument(0), invocation.getArgument(1), invocation.getArgument(2));
-            return null;
-        }).when(authenticateNetworkImplMock).getUser(ArgumentMatchers.any(String.class), ArgumentMatchers.any(String.class),
-                ArgumentMatchers.any(DataLoadListener.class));
-    }
-
-
-    private void handleUpdateDataResponse(Object result, DataLoadListener listener){
-        if(listener != null ){
-            listener.onDataLoad(getMockedUser());
-        }else{
-            assertFalse(false);
-        }
-    }
-
-    private void setUpMockedAuthNetworkDaoMethods() {
-        Mockito.doAnswer((Answer) invocation -> {
-            handleUpdateDataResponse(invocation.getArgument(0),invocation.getArgument(1));
-            return null;
-        }).when(authenticateDAOMock).updateDataForRegisterUser(ArgumentMatchers.any(Object.class),
-                ArgumentMatchers.any(DataLoadListener.class));
-
-        Mockito.when(authenticateDAOMock.getUser()).thenReturn(getMockedUser());
-    }
-
-    @After
-    public void tearDown() {
-        dbhelper.close();
-    }
+public class AuthenticateManagerImplTest extends YonaTestCase
```

Comment:
I see I didn't reformat the tests. Sorry about that. Can you do a separate PR with a reformat of the remaining tests?

Reply:
ok. will do that